lib/node/NodeTargetPlugin.js

```diff
@@ -5,61 +5,14 @@
 
 "use strict";
 
+const { builtinModules } = require("module");
 const ExternalsPlugin = require("../ExternalsPlugin");
 
 /** @typedef {import("../Compiler")} Compiler */
 
 const builtins = [
-	"assert",
-	"async_hooks",
-	"buffer",
-	"child_process",
-	"cluster",
-	"console",
-	"constants",
-	"crypto",
-	"dgram",
-	"diagnostics_channel",
-	"dns",
-	"dns/promises",
-	"domain",
-	"events",
-	"fs",
-	"fs/promises",
-	"http",
-	"http2",
-	"https",
-	"inspector",
-	"module",
-	"net",
-	"os",
-	"path",
-	"path/posix",
-	"path/win32",
-	"perf_hooks",
-	"process",
-	"punycode",
-	"querystring",
-	"readline",
-	"repl",
-	"stream",
-	"stream/promises",
-	"stream/web",
-	"string_decoder",
-	"sys",
-	"timers",
-	"timers/promises",
-	"tls",
-	"trace_events",
-	"tty",
-	"url",
-	"util",
-	"util/types",
-	"v8",
-	"vm",
-	"wasi",
-	"worker_threads",
-	"zlib",
+	...builtinModules,
+
```

Comment:
@vetruvet  Is there still a need to add a wasi module, which is not currently in builtinModules?
https://nodejs.org/api/wasi.html

```suggestion

	"wasi",

```

Reply:
I checked on Node.js 12, 14, 16, and 18, and they all have `wasi` included in `builtinModules`, even without the `--experimental-wasi-unstable-preview1` flag.

Reply:
I use Node.js 19.8.1

Reply:
Ah, it seems in Node.js 19 (and moving forward, presumably), `builtinModules` only includes `wasi` if node is run with that flag... Which further pokes a hole in this approach if the goal is for this list to include all of the latest Node.js modules, regardless of the Node.js version/flags webpack itself is running under.